spec/spec_helper.rb

```diff
@@ -37,6 +34,8 @@
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  config.filter_run_excluding disabled: true
```

Comment:
This allows a :disabled tag to be added to tests

Reply:
Do we still need this though?
Seems you have moved all potentially disable-able tests out into another PR

Reply:
Correct, but an automatically-applied `disabled` tag can still be useful for local dev I think